.azure-pipelines/release.yml

```diff
@@ -0,0 +1,85 @@
+trigger:
+  tags:
+    include:
+      - '*.*.*'
+
+pr: none
+
+stages:
+
+  - stage: 'build_conda_package'
+    displayName: 'Build Conda Package'
+    jobs:
+      - job: 'linux'
+        displayName: 'Linux'
+        timeoutInMinutes: 120
+        strategy:
+          matrix:
+            Python37:
+              PYTHON_VERSION: '3.7'
+        pool:
+          vmImage: 'ubuntu-latest'
+        variables:
+          - group: tokens
+        steps:
+          - checkout: self
+            submodules: true
+          - bash: | 
+              echo "##vso[task.prependpath]$CONDA/bin"
+              echo "##vso[task.setvariable variable=conda_dir]$CONDA"
+            displayName: 'add conda to PATH'
+          - bash: |
+              set -ex
+              conda --version
+              conda install --yes anaconda-client conda-build
+              conda config --set always_yes yes --set changeps1 no
+            displayName: 'Conda configuration'
+          - bash: |
+              conda build --user scipp --token "$ANACONDA_TOKEN" --channel conda-forge --label dev --python="$PYTHON_VERSION" .
```

Comment:
@nvaytet do you think we should change this label to be main? I've deliberately put  the packages off to the side while we test them, but it's more of a headache for users to get the channels right.

Reply:
yes, change to `main`

Reply:
Agreed another thing this will allow if for developers on osx to use `scipp-developer.yaml` as is.